tests/unit/test_network_connections.py

```diff
@@ -2912,49 +2941,51 @@ def test_full_state_absent_defaults(self):
             },
         )
 
-    def _test_ethtool_changes(self, input_features, expected_features):
+    def _test_ethtool_changes(self, input_ethtool, expected_ethtool):
         """
         When passing a dictionary 'input_features' with each feature and their
         value to change, and a dictionary 'expected_features' with the expected
         result in the configuration, the expected and resulting connection are
         created and validated.
         """
-        custom_ethtool_features = copy.deepcopy(ETHTOOL_FEATURES_DEFAULTS)
-        custom_ethtool_features.update(expected_features)
-        expected_ethtool = {"features": custom_ethtool_features}
-        input_connection = {
-            "ethtool": {"features": input_features},
+        expected_ethtool_full = copy.deepcopy(ETHTOOL_DEFAULTS)
+        for key in list(expected_ethtool_full):
+            if key in expected_ethtool:
+                expected_ethtool_full[key].update(expected_ethtool[key])
+
+        input_conn = {
+            "ethtool": input_ethtool,
             "name": "5",
             "persistent_state": "present",
             "type": "ethernet",
         }
 
-        expected_connection = {
+        expected_conn = {
             "actions": ["present"],
-            "ethtool": expected_ethtool,
+            "ethtool": expected_ethtool_full,
             "interface_name": "5",
             "persistent_state": "present",
             "state": None,
             "type": "ethernet",
         }
-        self.check_one_connection_with_defaults(input_connection, expected_connection)
+        self.check_one_connection_with_defaults(input_conn, expected_conn)
```

Comment:
Why are the variables abbreviated?

Reply:
I think it was to not break the 80 column limit. Otherwise, you'd have to split the line (I remember black was complaining).

Reply:
black will just split the line if you run it locally, so that would not be a problem.

Reply:
install black first, `pip install black`. Then run cmd to make the magic happens, `black test_network_connections.py --line-length 79`.

Reply:
IIRC, the system roles use the black default line lenght (88 chars)